src/modules/raids/sagas.js

```diff
@@ -0,0 +1,23 @@
+import { call, put, take } from 'redux-saga/effects';
+
+import { FETCH_RAIDS, loadRaids } from './actions';
+import { getRaids } from './services';
+
+export function* watchLoadRaids() {
+  while (true) {
+    yield take(FETCH_RAIDS);
+    yield call(loadRaidSaga);
+  }
+}
+
+export function* loadRaidSaga() {
+  try {
+    const raids = yield call(getRaids);
+    console.log('Fetch successful!');
+    console.log(raids.data);
+    yield put(loadRaids(raids.data));
+  }
+  catch (error) {
+    console.log(error)
```

Comment:
Later, we will need to display some sort of error when the request fails, that putting it here in `catch` would be a good place. What do you think? Something like `yield put(displaySnackbar('Error Message'));`